test/compression.js

```diff
@@ -664,6 +692,28 @@ function createServer (opts, fn) {
   })
 }
 
+function createHttp2Server (opts, fn) {
+  var _compression = compression(opts)
+  var server = http2.createServer(function (req, res) {
+    _compression(req, res, function (err) {
+      if (err) {
+        res.statusCode = err.status || 500
+        res.end(err.message)
+        return
+      }
+
+      fn(req, res)
+    })
+  })
+  server.listen(8443, '127.0.0.1')
```

Comment:
lol, I ran this on my machine and it failed because the port is hard-coded to a common port. Can this port be dynamic so it'll just always work without needing to stop other running services?

Reply:
Done.